test/playwright/provider/geolonia/e2e.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { test, expect } from '@playwright/test'
+import { test, expect } from '@playwright/test/index.mjs'
```

Comment:
how about dropping support to geolonia too, and make charites only focus on maplibre? 

It does not make sense to me to remove mapbox and keep geolonia.

Reply:
I don't care about it because Geolonia still uses Charites.

How about you in this comment? > @keichan34 @naogify

Reply:
@smellman There are possible 4 cases:

1. geolonia is compatible with maplibre, but not maplibre.
2. geolonia is compatible with mapbox, but not maplibre.
3. geolonia is compatible with both mapbox and maplibre
4. geolonia is not compatible with either mapbox or maplibre.

If it is case 1, it does not need to support geolonia. they can use charites as maplibre. No need to differentiate geolonia and maplibre. it is same thing.
if it is case 2, geolonia should be removed since mapbox is removed in this PR.
If it is case 3, geolonia can be removed from charites, but functionality is limited (not supporting mapbox featuer).
If it is case 4, it requires more efforts on third parties to support incompatible map library with maplibre.

Reply:
@JinIgarashi geolonia is an extension of maplibre, so that probably falls under case 1 (the parts we add are not really around styling). I'll double check our workflows to see if relying on vanilla maplibre works for us.

Reply:
@keichan34 How about current status? I want to release until Sunday.

Reply:
@smellman Sorry about the late response. Yeah, let's drop geolonia. That leaves just maplibre, so I guess we can get rid of the provider option altogether?

Reply:
@keichan34 Thank you for response. Yes, and we can drop $HOME/.charites/config.yml support too.